src/components/Layout/index.tsx

```diff
@@ -62,6 +62,9 @@ const ClientOnly: React.FC<any> = ({ children, ...delegated }) => {
   return hasMounted ? <div {...delegated}>{children}</div> : null;
 };
 
+const {GATSBY_MTM_SITE_ID} = process.env;
```

Comment:
Should these be destructured? There was a removed comment around them returning empty values.